.idea/compiler.xml

```diff
@@ -22,11 +22,12 @@
         <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
         <outputRelativeToContentRoot value="true" />
         <processorPath useClasspath="true" />
+        <module name="gwtp-idea" />
       </profile>
     </annotationProcessing>
     <bytecodeTargetLevel target="1.6">
+      <module name="gwtp-idea" target="1.7" />
       <module name="plugin-templates" target="1.6" />
     </bytecodeTargetLevel>
   </component>
-</project>
-
+</project>
```

Comment:
Yeah I know, I'm having a hard time removing it without deleting locally.. Tried `git rm --cached -r .idea`. It removes i from git locally, but the push doesn't seem to take it into account